addon-test-support/-private/ember-exam-qunit-test-loader.js

```diff
@@ -118,6 +119,17 @@ export default class EmberExamQUnitTestLoader extends TestLoader {
     super.unsee(moduleName);
   }
 
+  /**
+   * setupModuleMetadataHandler() register QUnit callback to enable generating module metadata file.
+   */
+  setupModuleMetadataHandler() {
+    this._qunit.moduleDone((metadata) => {
+      // testem:module-done-metadata is sent to server to keep track of test module details.
+      // 'metadata' contains module name, total number of assertion ran in the module, and module runtime.
+      this._testem.emit('testem:module-done-metadata', metadata);
```

Comment:
this line is breaking our test runs right now, saying:

> Cannot read property 'emit' of undefined

are you using a newer `testem` version? I'm not sure where `this._testem` is coming from 🤔

Reply:
this._testem is coming from https://github.com/ember-cli/ember-exam/pull/265/files#diff-bcd6d6c5d835e949677ec46d11a608dbR17.

When instantiating EmberExamQUnitTestLoader a testem object is passed
https://github.com/ember-cli/ember-exam/blob/master/addon-test-support/load.js#L20

Reply:
I'm seeing a similar thing in two apps after upgrading ember-exam

Running http://localhost:4200/tests

Reply:
@Turbo87 @BarryThePenguin Hi all, the issue with v4.0.3 is that testem object is not available during `ember serve`. The fix was merged: https://github.com/ember-cli/ember-exam/pull/377

please downgrade ember-exam to v4.02 or upgrade to v4.0.4 ( which is a version include the fix), Thanks!

Reply:
Great, thanks for getting back to us! 😊